.github/labeler.yml

```diff
@@ -20,6 +20,19 @@
         - scripts/**/*
         - win32/build/**/*
 
+"Category: CI":
+  - changed-files:
+      - any-glob-to-any-file:
+          - .circleci/*
+          - .github/actions/**/*
+          - .github/CODEOWNERS
```

Comment:
If we consider this to be CI, we might as well consider anything in `.github/` to be CI. The only exceptions I'm seeing right now are `lsan-suppressions.txt` and `labeler.yml` itself.

Reply:
I was trying to exclude `ISSUE_TEMPLATE` folder and potentially future files like funding, code of conduct, etc files that are merely templates.